core/store/migrations/0016_pipeline_task_run_dot_id.go

```diff
@@ -0,0 +1,48 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+const (
+	up16 = `
+ALTER TABLE pipeline_task_runs ADD COLUMN dot_id text; 
+UPDATE pipeline_task_runs SET dot_id = ts.dot_id FROM pipeline_task_specs ts WHERE ts.id = pipeline_task_runs.pipeline_task_spec_id;
+ALTER TABLE pipeline_task_runs ALTER COLUMN dot_id SET NOT NULL; 
+
+ALTER TABLE pipeline_task_runs ALTER COLUMN pipeline_task_spec_id DROP NOT NULL; 
+ALTER TABLE pipeline_task_runs DROP CONSTRAINT pipeline_task_runs_pipeline_task_spec_id_fkey;
+ALTER TABLE pipeline_task_runs DROP COLUMN pipeline_task_spec_id;
```

Comment:
I think the previous two are unnecessary, dropping column will automatically remove constraints and it doesn't matter about not nulls

Reply:
`ALTER TABLE pipeline_task_runs ALTER COLUMN dot_id SET NOT NULL,  DROP COLUMN pipeline_task_spec_id;`